packages/rpc-subscriptions-spec/README.md

```diff
@@ -40,7 +40,7 @@ try {
 
 ### `RpcSubscriptionsChannel<TOutboundMessage, TInboundMessage>`
 
-A channel is a `DataPublisher` that you can subscribe to events of type `RpcSubscriptionChannelEvents<TInboundMessage>`. Additionally, you can use it to send messages of type `TOutboundMessage` back to the remote end by calling the `send(message)` method.
+A channel is a `DataPublisher` on which you can subscribe to events of type `RpcSubscriptionChannelEvents<TInboundMessage>`. Additionally, you can use this object to send messages of type `TOutboundMessage` back to the remote end by calling its `send(message)` method.
```

Comment:
I decided not to complete the `README` here with all the new documentation in this PR. I think after this is all done we just delete all the READMEs and replace them with a link to the live documentation, whether that's on our site or on tsdocs.dev.